packages/borapark-vanilla/utils.test.js

```diff
@@ -0,0 +1,44 @@
+import { formatCurrencyAsLocaleString } from "./utils.js";
+
+describe("formatCurrencyAsLocaleString()", () => {
+  test("숫자를 전달하면, 세 자리마다 쉼표를 추가한 문자열로 반환한다.", () => {
+    expect(formatCurrencyAsLocaleString(1_000_000)).toBe("1,000,000");
+    expect(formatCurrencyAsLocaleString(384_729_343)).toBe("384,729,343");
+    expect(formatCurrencyAsLocaleString(857_295_720_374_230)).toBe(
+      "857,295,720,374,230"
+    );
+  });
+
+  test("0을 전달하면 '0'을 반환한다.", () => {
+    expect(formatCurrencyAsLocaleString(0)).toBe("0");
+  });
+
+  test("음수를 전달하면 예외가 발생한다.", () => {
+    expect(() => formatCurrencyAsLocaleString(-1000)).toThrow();
+  });
+
+  test("소수점을 포함한 숫자를 전달하면 예외가 발생한다.", () => {
+    expect(() => formatCurrencyAsLocaleString(1234.56)).toThrow();
+  });
+
+  test("NaN을 전달하면 예외가 발생한다.", () => {
+    expect(() => formatCurrencyAsLocaleString(NaN)).toThrow();
+  });
+
+  test("null을 전달하면 예외가 발생한다.", () => {
+    expect(() => formatCurrencyAsLocaleString(null)).toThrow();
+  });
+
+  test("undefined를 전달하면 예외가 발생한다.", () => {
+    expect(() => formatCurrencyAsLocaleString(undefined)).toThrow();
+  });
+
+  test("숫자 형태의 문자열을 전달하면 예외가 발생한다.", () => {
+    expect(() => formatCurrencyAsLocaleString("1000")).toThrow();
+  });
```

Comment:
minor: 하나하나 나누셔도 상관은 없지만, 이 정도로 단순한 비교라면 `0 또는 양수가 아닌 값을 전달하면 예외가 발생한다`로 정리했어도 될 것 같습니다. 그리고 for 문을 사용해서 나중에라도 추가하기 쉽게 작성하는 방법도 있습니다. 필수는 아닙니다. 😄 
```suggestion
  test("0 또는 양의 정수가 아닌 값을 전달하면 예외가 발생한다", () => {
    const disallowed = [-1000, 1234.56, NaN, null, undefined, '1000'];
    for (const value of disallowed) {
          expect(() => formatCurrencyAsLocaleString(value)).toThrow();
    }
  });
```

Reply:
이후 PR에서 피드백 반영할 수 있도록 하겠습니다!